content/attacks/posts/2022-01-17-Crypto.com.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2022-01-17
+custodians: Crypto.com
+categories: Custodian
+title: Crypto.com Confirms $34M Hack
+---
+
+## Summary
+
+According to the announcement, an attacker found a way to approve transactions without the two-factor authentication control being inputted by account holders. This resulted in 483 Crypto.com users losing funds from their accounts. The exchange reaffirmed comments made by the firm�s CEO, Kris Marszalek, that any accounts found to be impacted were fully restored, resulting in no loss of funds for users.
+
+## Attackers
+
+The identity of the attackers remained unknown.
+
+## Losses
+
+Attacker drained 4,836.26 ETH, 443.93 BTC, and approximately $66,200 of other currencies from its users� accounts. The stolen funds total approximately $34.4 million.
+
+## Timeline
+
+On Monday, Jan. 17, at approximately 00:46 UTC, the exchange�s risk monitoring systems detected unauthorized activity on a small number of user accounts.
```

Comment:
add additional timeline details related to when exchange paused withdrawls/access, how long, for how many users, etc.

Reply:
Done